controllers/scripts/ipfs.go

```diff
@@ -233,17 +228,9 @@ func createTemplateConfig(
 	identity, err = config.CreateIdentity(os.Stdout, []options.KeyGenerateOption{
 		options.Key.Type(options.Ed25519Key),
 	})
-	// fall back to RSA
 	if err != nil {
-		identity, err = config.CreateIdentity(os.Stdout, []options.KeyGenerateOption{
-			options.Key.Type(options.RSAKey),
-			options.Key.Size(4096),
-		})
-		if err != nil {
-			return
-		}
+		return
```

Comment:
We should return the error though?

Reply:
@hsanjuan This is just a named return, the error `err` (declared on line 223) will be returned automatically, along with an empty `config.Config` object, since nothing was assigned to `conf`.

Reply:
ah, of course...

Reply:
looks good, you can merge at will